packages/gawk.rb

```diff
@@ -37,21 +25,23 @@ def self.patch
   end
 
   def self.build
```

Comment:
Maybe leave in `system '[ -x configure ] || autoreconf -fvi` for when we do build from tarball?

Reply:
we are building from a tarball, but the git repository has configure in it anyway? I'm not sure why.